apps/dot-voting/app/components/VotingOption.js

```diff
@@ -60,11 +60,18 @@ const VotingOption = ({ valueSpring, label, percentage, color, threshold, userVo
             )}
           </div>
         )}
-        {percentage !== -1 &&
-            <Text size="xsmall" color="#98A0A2">
-              {Math.round(percentage)}%
+        <div>
+          {percentage !== -1 &&
+              <Text size="xsmall" color="#98A0A2">
+                {Math.round(percentage)}%
+              </Text>
+          }
+          {allocation && (
+            <Text size="xsmall" color="#98A0A2" css={`margin-left: ${0.25 * GU}px`}>
+                ({allocation})
```

Comment:
Can we use theme for the colors, please?

Reply:
Done. I also noticed another component that used a hardcoded colour so I changed that as well.

Reply:
@PeterMPhillips, that way of importing `theme` is deprecated. The new way of doing it is by importing the `useTheme` hook,

    import { useTheme } from '@aragon/ui'

and then, inside a functional component,

     const theme = useTheme()

Reply:
I tried that. But useTheme() doesn't pass any text values, and I need textTertiary. This is probably why it was hardcoded in the first place.